src/content.rs

```diff
@@ -204,14 +210,56 @@ impl Content {
             published,
         }
     }
+    pub fn load_shortcode_dir(
```

Comment:
There seems to be some inconsistency between `shortcode` and `shortcodes` in various documentation and identifier names.